### PR TITLE
Feature UI client

### PR DESCRIPTION
**Pull request by SaulCM**
Added a request creator and audio parameters form

### CODE REVIEW

File: logs/propertieslogs.log [hunk 1]
```diff
@@ -126,3 +126,19 @@
...
+[INFO ] 2021-04-13 17:51:03.324 [main] ConverterApplication - finish
+[INFO ] 2021-04-13 17:51:28.913 [http-nio-8080-exec-1] ExtractTextController - start
+[INFO ] 2021-04-13 17:51:28.915 [http-nio-8080-exec-1] ExtractText - start
+[INFO ] 2021-04-13 17:51:28.917 [http-nio-8080-exec-1] ExtractText - Execute Try
+[INFO ] 2021-04-13 17:51:29.261 [http-nio-8080-exec-1] ExtractText - start
+[INFO ] 2021-04-13 17:51:29.261 [http-nio-8080-exec-1] ExtractText - Execute Try
+[INFO ] 2021-04-13 17:51:29.265 [http-nio-8080-exec-1] ExtractText - finish
+[INFO ] 2021-04-13 17:51:29.265 [http-nio-8080-exec-1] ExtractText - finish
+[INFO ] 2021-04-13 17:51:29.271 [http-nio-8080-exec-1] ExtractTextController - finish
+[INFO ] 2021-04-13 22:51:40.358 [main] ConverterApplication - start
```
Comment (jpgonzales27) [inline logs/propertieslogs.log]: the log file should not be uploaded

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    public void setVolumeValue(String volumeValue) {
+        this.volumeValue = volumeValue;
+    }
+
+    public String getHzValue() {
+        return hzValue;
+    }
+
+    public void setHzValue(String hzValue) {
+        this.hzValue = hzValue;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java]: the javadoc should be put in the methods

File: src/main/java/org/fundacion/jala/converter/view/Models/IrequestForm.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
+ * @version 1.0
+ */
+
+public interface IrequestForm {
+
+}
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/Models/IrequestForm.java]: I think this empty interface should be deleted

File: src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java [hunk 4]
```diff
@@ -0,0 +1,114 @@
+package org.fundacion.jala.converter.view.Controller;
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java:1]: Package in under Copyrigth

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+    /**
+     * Audio Request Form stores parameters for an audio request
+     * @param filepathValue
+     * @param formatValue
+     * @param bitrateValue
+     * @param volumeValue
+     * @param hzValue
+     */
+    public AudioRequestForm(String filepathValue, String formatValue, String bitrateValue, String volumeValue, String hzValue) {
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java]: Add parameter String metadata

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    /**
+     * Add volume parameter
+     * @param volumeValue
+     */
+    public void addVolume(String volumeValue) {
+        addParameters(new Parameter(VOLUME, volumeValue, false));
+    }
+
+    /**
+     * Add hz parameter
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java:72]: I think this comment must be in the third person

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+     * @param hzValue
+     */
+    public void addHz(String hzValue) {
+        addParameters(new Parameter(HZ, hzValue, false));
+    }
+
+    /**
+     * @return bodyParameters
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java:80]: Is missing a description of the method

**Review by GustavoZHA — APPROVED** [review]

**Review by danielaSantaCruz — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
+ * @version 1.0
+ */
+
+public class AudioRequestForm implements IrequestForm{
+    public List<Parameter> bodyParameters = new ArrayList<>();
+    public final String URL = "http://localhost:8080/api/extractText";
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java:20]: Change URL extractText to convertAudio

**Review by edsonarios — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Review by Paola-Xaq — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request closed without merging**